index.html

```diff
@@ -1913,6 +1919,64 @@
 
 </aside>
 
+## Transparency
+
+<div class="practice">
+  <span class="practicelab" id="transparency-when-requested">
+    [=People=] should be provided relevant information when data is accessed or when permission is
+    requested.
```

Comment:
This feels vague. Which people? What information is relevant? (That's covered below, but it'd be good if the principle could be read alone.) Should we interrupt people whenever a backend system touches their data?

I think we have a couple goals in this section:

1. People should be made aware of (or be able to find out?) what actors have their data and how those actors use it.
2. The above information should be in a form that allows people to automate their reactions to it.
3. Researchers and regulators should be able to aggregate that information (in privacy-preserving ways) in order to analyze how data is used and abused across society.

Maybe the first one would be covered by:

```suggestion
    [=People=] should be able to find out which [=actors=] have their data and how they're using it.
```

On the permissions point, maybe there's something more general about UAs needing to inform users when they give out information about the user, which pushes for designing web APIs that avoid passive fingerprintability and that provide an opportunity to ask the user's permission before disclosing the information.

Reply:
This text is broad (perhaps still too broad) in part because the intent is to go beyond discoverability about who has data about you, though that is good to call out separately as well.

Sites need to provide, and UAs may need to help present or consume, relevant information about data collection and use when collection is happening. Permissions is a special case of that where the APIs are easier to reason about, but sites also collect data in the course of navigation and in using HTML functionality (forms, etc.).